spec/errors/extend/placeholder/missing.hrx

```diff
@@ -11,6 +11,13 @@ Error: ".baz" failed to @extend "%foo".
         on line 2 of input.scss
   Use --trace for backtrace.
 
+<===> error-libsass
+Error: The target selector was not found.
+       Use "@extend %foo !optional" to avoid this error.
+        on line 2 of input.scss
+>>     @extend %foo;
+   ------------^
```

Comment:
This test, and a bunch of the others, now have an `error` output that doesn't actually match *any* implementation's error output. Please update them so that the Dart Sass error message is the canonical error.